api/v1/submissions/PKPSubmissionHandler.inc.php

```diff
@@ -312,6 +312,10 @@ public function add($slimRequest, $response, $args) {
 			return $response->withStatus(400)->withJsonError('api.submissions.403.contextRequired');
 		}
 
+		if (!$request->getContext()->getData('enableSubmissions')) {
+			return $response->withStatus(403)->withJsonError('author.submit.notAccepting');
+		}
```

Comment:
This locale file hasn't been loaded yet in the API handler so you'll need to load it with `AppLocale::requireComponents()`.